apps/dapp-console-api/src/testhelpers/auth.ts

```diff
@@ -66,10 +66,17 @@ export const validSession = async () => {
     envVars.PRIVY_ACCESS_TOKEN_SALT,
   )
 
-  return mockUserSession({
-    entityId: ENTITY_ID,
-    privyAccessTokenExpiration: Date.now() + 1000,
-    privyAccessToken: hashedAccessToken,
-    privyDid: PRIVY_DID,
-  })
+  return mockUserSession(
+    {
+      entityId: ENTITY_ID,
+      privyAccessTokenExpiration: Date.now() + 1000,
+      privyAccessToken: hashedAccessToken,
+      privyDid: PRIVY_DID,
+    },
+    {
+      nullifierHash: 'nullifierHash',
+      isVerified: true,
+      createdAt: Date.now().toString(),
+    },
```

Comment:
can you make this an argument that is passed in instead of setting it inside `validSession`? `validSession` is meant to just be a valid privy session, and then we should add additional arguments to the function that can be passed in in order to add other objects to the session.

```
export const validSession = async (worldIdSession?: WorldIdSessionType) => {
```